fs-core/main/default/lwc/formulaShareRulesListView/__tests__/formulaShareRulesListView.test.js

```diff
@@ -0,0 +1,155 @@
+import { createElement } from 'lwc';
+import { ShowToastEventName } from 'lightning/platformShowToastEvent';
+
+import FormulaShareRulesListView from 'c/formulaShareRulesListView';
+import { registerLdsTestWireAdapter } from '@salesforce/sfdx-lwc-jest';
+import { registerApexTestWireAdapter } from '@salesforce/sfdx-lwc-jest';
```

Comment:
Looks like this is called in line 10, but the parameter isn't used - think we can remove lines 6 and 10

Reply:
@LawrenceLoz 
Line 6 is removed but line 10 is already in use.

Reply:
When I remove line 10 a test is failing.

Reply:
Cool sounds good, thanks for clarifying